Containerfile

```
@@ -0,0 +1,16 @@
+FROM debian
+
+RUN apt-get update -y
+
+RUN apt-get install -y autoconf xcb-proto libxcb-keysyms1-dev glibc-source libjson-glib-dev gtk-doc-tools gobject-introspection libtool libx11-dev git
+
+RUN git clone https://github.com/altdesktop/i3ipc-glib/ \
+    && cd i3ipc-glib \
+    && ./autogen.sh \
+    && make \
+    && make install
```

This one seems to work:

```suggestion
    && make prefix=/usr install
```

It does not seem to look for libraries inside `/usr/local/lib`, so the above makes it write to `/usr/lib`. Might alternatively work to set the `LIBRARY_PATH` (or similar).